src/features/activities/reactions/ReactionCounts.tsx

```diff
@@ -25,7 +25,6 @@ export const ReactionCounts = ({
             color="reaction"
             data-myreaction={!!rg.myReaction}
             key={rg.reaction}
-            myReaction={rg.myReaction}
```

Comment:
@CryptoGraffe I couldn't find any use for this, and it caused a react error, but curious if you recall a reason for it.